src/reference/config/bindings/ws/.partials/server.yaml

```
@@ -0,0 +1,8 @@
+  ws_server:
+    type: ws
+    kind: server
+    exit: default_server
+    routes:
+      - when:
+          - protocol: echo
+        exit: echo_server
```

```suggestion
  ws_server:
    type: ws
    kind: server
    exit: echo_server
    routes:
      - when:
          - protocol: mqtt
        exit: mqtt_server
```

FYI - the MQTT specification requires specifying `mqtt` as the subprotocol over WebSocket, so this might be a helpful example based on standards. [[MQTT-6.0.0-3]](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901285)